kegtab/src/main/java/org/kegbot/api/KegbotApiImpl.java

```diff
@@ -49,6 +49,7 @@
 import org.kegbot.app.util.Version;
 import org.kegbot.backend.Backend;
 import org.kegbot.backend.BackendException;
+import org.kegbot.core.ThermoSensor;
```

Comment:
This is an unused import.

You should `import org.kegbot.proto.Models.ThermoSensor` instead. Then you can change the references from `Models.ThermoSensor` to just `ThermoSensor`.